flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -447,6 +447,43 @@ var A320_Neo_UpperECAM;
                             }
                         ]
                     },
+                    {
+                        name: "ENG",
+                        messages: [
+                            {
+                                id: "thr_levers_not_set_flex",
+                                message: "THR LEVERS NOT SET",
+                                level: 2,
```

Comment:
Inhibited in these flight phases:

```suggestion
                                level: 2,
                                flightPhasesInhib: [1, 4, 5, 6, 7, 8, 10],
```